mapped 64256 bytes (62 KB) for 1 cores
*** Operational MODE: single process ***
added /var/www/lab_app/ to pythonpath.
Traceback (most recent call last):
  File "/var/www/lab_app/hello.py", line 1, in <module>
    from flask import Flask
ImportError: No module named 'flask'
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI worker 1 (and the only) (pid: 2797, cores: 1)
--- no python application found, check your startup logs for errors ---
[pid: 2797|app: -1|req: -1/1] 192.168.1.102 () {42 vars in 762 bytes} [Mon Feb  1 14:38:46 2021] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 2797|app: -1|req: -1/2] 192.168.1.102 () {40 vars in 676 bytes} [Mon Feb  1 14:38:46 2021] GET /favicon.ico => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 2797|app: -1|req: -1/3] 192.168.1.102 () {42 vars in 762 bytes} [Mon Feb  1 14:38:52 2021] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 2797|app: -1|req: -1/4] 192.168.1.102 () {40 vars in 676 bytes} [Mon Feb  1 14:38:52 2021] GET /favicon.ico => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
*** Starting uWSGI 2.0.12 (32bit) on [Mon Feb  1 14:39:54 2021] ***
compiled with version: 4.9.2 on 15 August 2017 11:45:19
os: Linux-5.4.51-v7l+ #1333 SMP Mon Aug 10 16:51:40 BST 2020
nodename: raspberrypi
machine: armv7l
clock source: unix
pcre jit disabled
detected number of CPU cores: 4
current working directory: /var/www/lab_app
detected binary path: /var/www/lab_app/bin/uwsgi
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
*** WARNING: you are running uWSGI without its master process manager ***
your processes number limit is 60943
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /var/www/lab_app/lab_app_uwsgi.sock fd 3
Python version: 3.4.4 (default, Aug 20 2019, 08:42:52)  [GCC 8.3.0]
PEP 405 virtualenv detected: /var/www/lab_app
Set PythonHome to /var/www/lab_app
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x9b7cc0
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 64256 bytes (62 KB) for 1 cores
*** Operational MODE: single process ***
added /var/www/lab_app/ to pythonpath.
Traceback (most recent call last):
  File "/var/www/lab_app/hello.py", line 1, in <module>
    from flask import Flask
ImportError: No module named 'flask'
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI worker 1 (and the only) (pid: 2929, cores: 1)
--- no python application found, check your startup logs for errors ---
[pid: 2929|app: -1|req: -1/1] 192.168.1.102 () {42 vars in 762 bytes} [Mon Feb  1 14:39:59 2021] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 2929|app: -1|req: -1/2] 192.168.1.102 () {40 vars in 676 bytes} [Mon Feb  1 14:39:59 2021] GET /favicon.ico => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 2929|app: -1|req: -1/3] 192.168.1.102 () {42 vars in 762 bytes} [Mon Feb  1 14:40:01 2021] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 2929|app: -1|req: -1/4] 192.168.1.102 () {40 vars in 676 bytes} [Mon Feb  1 14:40:01 2021] GET /favicon.ico => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
*** Starting uWSGI 2.0.12 (32bit) on [Mon Feb  1 15:11:28 2021] ***
compiled with version: 4.9.2 on 15 August 2017 11:45:19
os: Linux-5.4.51-v7l+ #1333 SMP Mon Aug 10 16:51:40 BST 2020
nodename: raspberrypi
machine: armv7l
clock source: unix
pcre jit disabled
detected number of CPU cores: 4
current working directory: /var/www/lab_app
detected binary path: /var/www/lab_app/bin/uwsgi
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
*** WARNING: you are running uWSGI without its master process manager ***
your processes number limit is 60943
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /var/www/lab_app/lab_app_uwsgi.sock fd 3
Python version: 3.4.4 (default, Aug 20 2019, 08:42:52)  [GCC 8.3.0]
PEP 405 virtualenv detected: /var/www/lab_app
Set PythonHome to /var/www/lab_app
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0xe2fcc0
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 64256 bytes (62 KB) for 1 cores
*** Operational MODE: single process ***
added /var/www/lab_app/ to pythonpath.
Traceback (most recent call last):
  File "/var/www/lab_app/hello.py", line 1, in <module>
    from flask import Flask
ImportError: No module named 'flask'
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI worker 1 (and the only) (pid: 3408, cores: 1)
--- no python application found, check your startup logs for errors ---
[pid: 3408|app: -1|req: -1/1] 192.168.1.102 () {40 vars in 731 bytes} [Mon Feb  1 15:11:39 2021] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 3408|app: -1|req: -1/2] 192.168.1.102 () {40 vars in 676 bytes} [Mon Feb  1 15:11:39 2021] GET /favicon.ico => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
Gracefully killing worker 1 (pid: 3408)...
*** Starting uWSGI 2.0.12 (32bit) on [Fri Feb  5 18:48:30 2021] ***
compiled with version: 4.9.2 on 15 August 2017 11:45:19
os: Linux-5.4.51-v7l+ #1333 SMP Mon Aug 10 16:51:40 BST 2020
nodename: raspberrypi
machine: armv7l
clock source: unix
pcre jit disabled
detected number of CPU cores: 4
current working directory: /var/www/lab_app
detected binary path: /var/www/lab_app/bin/uwsgi
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
*** WARNING: you are running uWSGI without its master process manager ***
your processes number limit is 60943
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /var/www/lab_app/lab_app_uwsgi.sock fd 3
Python version: 3.4.4 (default, Aug 20 2019, 08:42:52)  [GCC 8.3.0]
PEP 405 virtualenv detected: /var/www/lab_app
Set PythonHome to /var/www/lab_app
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x15a0318
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 64256 bytes (62 KB) for 1 cores
*** Operational MODE: single process ***
added /var/www/lab_app/ to pythonpath.
Traceback (most recent call last):
  File "/var/www/lab_app/hello.py", line 1, in <module>
    from flask import Flask
ImportError: No module named 'flask'
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI worker 1 (and the only) (pid: 1523, cores: 1)
--- no python application found, check your startup logs for errors ---
[pid: 1523|app: -1|req: -1/1] 192.168.1.106 () {40 vars in 731 bytes} [Fri Feb  5 18:48:59 2021] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 1523|app: -1|req: -1/2] 192.168.1.106 () {40 vars in 676 bytes} [Fri Feb  5 18:48:59 2021] GET /favicon.ico => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
*** Starting uWSGI 2.0.12 (32bit) on [Fri Feb  5 19:06:45 2021] ***
compiled with version: 4.9.2 on 15 August 2017 11:45:19
os: Linux-5.4.51-v7l+ #1333 SMP Mon Aug 10 16:51:40 BST 2020
nodename: raspberrypi
machine: armv7l
clock source: unix
pcre jit disabled
detected number of CPU cores: 4
current working directory: /var/www/lab_app
detected binary path: /var/www/lab_app/bin/uwsgi
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
*** WARNING: you are running uWSGI without its master process manager ***
your processes number limit is 60943
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /var/www/lab_app/lab_app_uwsgi.sock fd 3
Python version: 3.4.4 (default, Aug 20 2019, 08:42:52)  [GCC 8.3.0]
PEP 405 virtualenv detected: /var/www/lab_app
Set PythonHome to /var/www/lab_app
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0xf59318
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 64256 bytes (62 KB) for 1 cores
*** Operational MODE: single process ***
added /var/www/lab_app/ to pythonpath.
Traceback (most recent call last):
  File "/var/www/lab_app/hello.py", line 4, in <module>
    from flask import Flask
ImportError: No module named 'flask'
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI worker 1 (and the only) (pid: 1711, cores: 1)
--- no python application found, check your startup logs for errors ---
[pid: 1711|app: -1|req: -1/1] 192.168.1.106 () {42 vars in 762 bytes} [Fri Feb  5 19:06:49 2021] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 1711|app: -1|req: -1/2] 192.168.1.106 () {40 vars in 676 bytes} [Fri Feb  5 19:06:50 2021] GET /favicon.ico => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 1711|app: -1|req: -1/3] 192.168.1.106 () {42 vars in 762 bytes} [Fri Feb  5 19:06:51 2021] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 1711|app: -1|req: -1/4] 192.168.1.106 () {40 vars in 676 bytes} [Fri Feb  5 19:06:51 2021] GET /favicon.ico => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 1711|app: -1|req: -1/5] 192.168.1.106 () {42 vars in 762 bytes} [Fri Feb  5 19:06:53 2021] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 1711|app: -1|req: -1/6] 192.168.1.106 () {40 vars in 676 bytes} [Fri Feb  5 19:06:53 2021] GET /favicon.ico => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 1711|app: -1|req: -1/7] 192.168.1.106 () {42 vars in 762 bytes} [Fri Feb  5 19:06:54 2021] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 1711|app: -1|req: -1/8] 192.168.1.106 () {40 vars in 676 bytes} [Fri Feb  5 19:06:54 2021] GET /favicon.ico => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
